build/win32/code.iss

```diff
@@ -1376,6 +1381,11 @@ begin
     Result := True;
 end;
 
+function IsWindows11OrLater(): Boolean;
+begin
+  Result := (GetWindowsVersion >= $0A0055F0);
```

Comment:
Please refer to `Pascal Scripting: GetWindowsVersion` section for how the version bits are aligned https://jrsoftware.org/ishelp/